plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -249,6 +260,32 @@ public function objectReplaced(BaseObject $object, BaseObject $replacingObject,
 		return true;
 	}
 
+	/* (non-PHPdoc)
+	 * @see kObjectCopiedEventConsumer::objectCopied()
+	 */
+	public function objectCopied(BaseObject $fromObject, BaseObject $toObject)
+	{
+		if($fromObject instanceof entry)
+		{
+			$c = new KalturaCriteria();
+			$c->add( CuePointPeer::ENTRY_ID, $fromObject->getId() );
+			$c->addAscendingOrderByColumn(CuePointPeer::CREATED_AT);
```

Comment:
Why does the order matters? You could use CuePointPeer::retrieveByEntryId instead.